src/test/java/com/easypost/AddressTest.java

```diff
@@ -0,0 +1,101 @@
+package com.easypost;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.List;
+
+import com.easypost.exception.EasyPostException;
+import com.easypost.model.Address;
+import com.easypost.model.AddressCollection;
+import com.easypost.model.Batch;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class AddressTest {
+    
+    public static Address address;
+
+    @BeforeClass
+    public static void setup() throws EasyPostException{
+        EasyPost.apiKey = "EASYPOST_TEST_API_KEY";
+        address = Address.create(Fixture.basicAddress());
+    }
+
+    @Test
+    public void testCreate() throws EasyPostException {
+        Address address = Address.create(Fixture.basicAddress());
+
+        assertNotNull(address);
+        assertEquals("388 Townsend St", address.getStreet1());
+        assertTrue(address.getId().startsWith("adr_"));
+    }
+
+    @Test
+    public void testCreateVerifiyStrict() throws EasyPostException {
+        Map<String, Object> addressData = Fixture.basicAddress();
+        List<String> verificationList = new ArrayList<>();
+        verificationList.add("true");
+        addressData.put("verify_strict", verificationList);
+
+        Address address = Address.create(addressData);
+
+        assertNotNull(address);
```

Comment:
Please follow the PHP library and assert that these are the correct instances. Asserting that an object is not null is not a good test when we immediately follow it up with asserting an exact value (such as street1 on the next line), because then obviously if that's true, the object couldn't have been null. Let's keep these as close to the PHP one as possible.